src/features/cellnav/js/cellnav.js

```diff
@@ -888,8 +888,12 @@
                   var values = [];
                   var currentSelection = grid.api.cellNav.getCurrentSelection();
                   for (var i = 0; i < currentSelection.length; i++) {
-                    var cellDisplayValue = getAppendedColumnHeaderText(currentSelection[i].col) + getCellDisplayValue(currentSelection[i]);
-                    values.push(cellDisplayValue)
+                    var cellValue = getCellDisplayValue(currentSelection[i]);
+                    if (cellValue === '') {
```

Comment:
why not just use if (cellValue)

Reply:
I don't want just any false-y value to get caught. Only empty strings. 

I suppose every value it will hold comes back as a string, but I'm not sure.

Reply:
FWIW in the other cases of false-y you probably want to do this. Specifically undefined which is not ''

Reply:
I was thinking of a cell that says 0 or is an uncheck checkbox or something. Neither of those are `Empty Cell`

Reply:
![image](https://user-images.githubusercontent.com/50843504/59122163-0fe6ec80-8928-11e9-9e01-137c0409aa83.png)


This is what I'm talking about.